src/terra-dev-site/tool/terra-dev-site/configPropsTable.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/* eslint-disable react/forbid-prop-types, react/no-unused-prop-types */
+const propTypes = {
```

Comment:
i threw this together to show terra-dev-site config as a props table instead of having to build out a table myself.